.pipelines/OneBranch.Official.yaml

```diff
@@ -0,0 +1,102 @@
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+  paths:
+    exclude:
+      - ".devcontainer"
+      - ".hooks"
+      - ".vscode"
+      - ".github"
+      - docs
+
+trigger:
+  branches:
+    include:
+      - gh-readonly-queue/master/*
+  tags:
+    include:
+      - "*"
+
+resources:
+  repositories:
+  - repository: templates
+    type: git
+    name: OneBranch.Pipelines/GovernedTemplates
+    ref: refs/heads/master
```

Comment:
I noticed within my work for other pipelines that it really helps to have this ref be parameterized so that you can test infra updates that are not pipeline steps before they merge to master.

Reply:
This is just a reference to the OB templates. They recommend this setting in their pipeline files. I don't think we will be using a different version. And if versioning becomes an issue, then we can pin it at that time.